server_admin/topics/sso-protocols/con-oidc-auth-flows.adoc

```diff
@@ -131,6 +131,13 @@ The CIBA grant uses the following two providers.
 
 {project_name} has both default providers. However, the administrator needs to set up Authentication Channel Provider like this:
 
+ifeval::["{kc_dist}" == "quarkus"]
+[source,bash,subs="attributes+"]
+----
+kc.[sh|bat] start --spi-ciba-auth-channel-ciba-http-auth-channel-http-authentication-channel-uri=https://backend.internal.example.com{kc_base_path}
+----
+endif::[]
+ifeval::["{kc_dist}" == "wildfly"]
```

Comment:
Strictly speaking since this is about configuring the server I would have preferred it in server config guide, or to be honest CIBA isn't really properly supported by Keycloak and requires customizing/extending so should perhaps have been in server dev guide. However, that's a too big change so okay to keep it here for now.